src/drivers/led/led.cpp

```diff
@@ -139,9 +139,5 @@ drv_led_start(void)
 {
 	if (gLED == nullptr) {
 		gLED = new LED;
-
-		if (gLED != nullptr) {
```

Comment:
Because it is already initialized on the LED constructor.
```
LED::LED() :
#ifdef __PX4_NUTTX
	CDev("led", LED0_DEVICE_PATH)
#else
	VDev("led", LED0_DEVICE_PATH)
#endif
{
	// force immediate init/device registration
	init();
}
```